python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -229,152 +292,188 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
-                fitParams["yMax"], fitParams["yMin"]], "k", alpha=0.3)
+        # Points used for the fit
+        fitPoints = fitParams["fitPoints"]
 
         # Add some useful information to the plot
-        bbox = dict(alpha=0.9, facecolor="white", edgecolor="none")
-        medMag = np.median(mags)
+        SNsUsedDict = {}
+        if hasattr(self.config.selectorActions, "catSnSelector"):
+            # Compute the effective S/N & mag cuts for the points used in the
+            # fit.
+            for SNBand in self.config.selectorActions.catSnSelector.bands:
+                SNsUsed = (catPlot[SNBand + "_" + plotInfo["SNFlux"]].values[fitPoints]
+                           / catPlot[SNBand + "_" + plotInfo["SNFlux"] + "Err"].values[fitPoints])
+                minSnUsed = np.nanmin(SNsUsed)
+                magsUsed = mags[fitPoints]
+                incr = 5.0
+                idsUsed = (SNsUsed < minSnUsed + incr)
+                while sum(idsUsed) < max(0.005*len(idsUsed), 3):
+                    incr += 5.0
+                    idsUsed = (SNsUsed < plotInfo["SN"] + incr)
+                medMagUsed = np.nanmedian(magsUsed[idsUsed])
+                SNsUsedDict[SNBand] = {"minSnUsed": minSnUsed, "medMagUsed": medMagUsed}
 
-        try:
-            SN = self.config.selectorActions.catSnSelector.threshold
-            SNBand = self.config.axisLabels["mag"][0]
-            SNFlux = self.config.selectorActions.catSnSelector.fluxType
-            SNs = catPlot[SNBand + "_" + SNFlux]/catPlot[SNBand + "_" + SNFlux + "Err"]
-            ids = (SNs < SN + 10.0)
-            medMag = np.nanmedian(mags[ids])
-        except AttributeError:
-            SN = "NA"
-            medMag = "NA"
-
-        infoText = "N Used: {}\nN Total: {}\nS/N cut: {}\n".format(len(fitPoints), len(catPlot), SN)
-        infoText += r"Mag $\lesssim$: " + "{:0.2f}".format(medMag)
-        ax.text(0.05, 0.78, infoText, color="k", transform=ax.transAxes,
-                fontsize=8, bbox=bbox)
+        bbox = dict(alpha=0.9, facecolor="white", edgecolor="none")
+        infoText = "N Total: {}".format(len(catPlot))
+        ax.text(0.04, 0.97, infoText, color="k", transform=ax.transAxes,
+                fontsize=7, bbox=bbox, va="top")
+        infoText = "N Used in fit: {}".format(sum(fitPoints))
+        ax.text(0.04, 0.925, infoText, color="darkblue", transform=ax.transAxes,
+                fontsize=7, bbox=bbox, va="top")
+        if len(SNsUsedDict) > 0:
+            yLoc = 0.9
+            for SNBand in self.config.selectorActions.catSnSelector.bands:
+                yLoc -= 0.04
+                infoText = "{}: S/N".format(SNBand)
+                infoText += r"$\geq$" + "{:0.1f} [".format(SNsUsedDict[SNBand]["minSnUsed"])
+                infoText += r"$\lesssim$ " + "{:0.1f} mag]".format(SNsUsedDict[SNBand]["medMagUsed"])
+                ax.text(0.04, yLoc, infoText, color="C0", transform=ax.transAxes,
+                        fontsize=6, va="center")
```

Comment:
I don't like the text and will try to come up with something neater for the future.